build.gradle

```diff
@@ -179,15 +179,12 @@ project.tasks['pitest'].group="verification"
 
 sonarqube {
   properties {
-    property "sonar.projectName", "Reform :: ccd-case-document-am-api"
```

Comment:
Please disable the sonar for now, as we cannot configure the limit and it will be enabled once coverage reaches 80%

Reply:
Done